methods/matching/calculate_k.py

```diff
@@ -66,10 +69,11 @@ def build_layer_collection(
 
     # ecoregions is such a heavy layer it pays to just rasterize it once - we should possibly do this once
     # as part of import of the ecoregions data
-    ecoregions = GroupLayer([
-        RasterLayer.layer_from_file(os.path.join(ecoregions_directory_path, filename)) for filename in
-            glob.glob("*.tif", root_dir=ecoregions_directory_path)
-    ], name="ecoregions")
+    # ecoregions = GroupLayer([
+    #     RasterLayer.layer_from_file(os.path.join(ecoregions_directory_path, filename)) for filename in
+    #         glob.glob("*.tif", root_dir=ecoregions_directory_path)
+    # ], name="ecoregions")
+    ecoregions = RasterLayer.layer_from_file(ecoregions_directory_path)
```

Comment:
@swinersha can you explain why you added this? Same with access.

Reply:
This looks like somehow @swinersha has based his work off the work-in-progress branch I have for per-project optimisation, rather than from main. But he doesn't seem have picked up any other changes oddly enough. I was worried I'd accidentally committed something I shouldn't to main, but that doesn't appear to be the case.